app/src/main/index.js

```diff
@@ -257,6 +258,12 @@ app.on(`ready`, () => createWindow())
 
 // start lcd REST API
 async function startLCD(home, nodeIP) {
+  assert.equal(
+    lcdProcess,
+    null,
+    `Can't start Gaia Lite because it's already running.  Call StopLCD first.`
```

Comment:
how do I call `StopLCD` ? though the Command line ? If so we'd need to add it to the README or add a bit more detail to the message.

 Also it should be called `StopGaiaLite` instead of `StopLCD`

Reply:
Sorry for the confusion; that error message is intended for developers.  It raises a good question:  Should we have a separate channel for developer errors vs. user errors?  Should the user error for this situation show something like, "An error has occurred.  Please restart Voyager and try again."

@faboweb Do we have any precedent for this?

Reply:
if it appears in a modal, it's a user error and should: 
1. be in plain english
2. provide an action for the user to move forward

if it appears in the console, it's for developers and should:
1. be in more detailed plain english
2. provide an action for the developer to move forward

:trollface:  🌮 👍

Reply:
I think it already satisfies the second case.

Reply:
@fedekunze does this satisfy your request for change?

Reply:
ok resolving then